main.yml

```diff
@@ -16,6 +16,14 @@
 
 # Primary entrypoint for all CDP Installations
 
+- name: Install required liraries
+  hosts: localhost
+  tags: always
+  gather_facts: yes
+  tasks:
+    - become: yes
+      ansible.builtin.pip:
+        name: boto3 >= 1.4.4
```

Comment:
Without this, a subsequent step failed because it is missing boto3.
Maybe there's a more elegant solution?

Reply:
Hmm, if this is purely local then we would not want it to need boto3 - possibly my implementation is faulty when the parcel loading tricks are used. Can you link me to the task that is failing?